docs/book/user-guide/advanced-guide/advanced-guide.md

```diff
@@ -6,6 +6,6 @@ description: Taking your ZenML workflow to the next level.
+<table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td><mark style="color:purple;"><strong>Pipelining features</strong></mark></td><td>Detailed look at how to configure and handle ZenML steps and pipelines.</td><td><a href="pipelining-features/pipelining-features.md">pipelining-features.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Data management</strong></mark></td><td>Managing your input and output data with ZenML.</td><td><a href="data-management/data-management.md">data-management.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Model management</strong></mark></td><td>Managing infrastructure.</td><td><a href="infrastructure-management/infrastructure-management.md">model-management.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Configuring ZenML</strong></mark></td><td>Setting up ZenML for your project.</td><td><a href="configuring-zenml/configuring-zenml.md">configuring-zenml.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Secret management</strong></mark></td><td>Registering and utilizing secrets with ZenML.</td><td><a href="secret-management/secret-management.md">secret-management.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Best practices</strong></mark></td><td>Learning how to best set up your ZenML projects.</td><td><a href="best-practices/best-practices.md">best-practices.md</a></td></tr></tbody></table>
```

Comment:
One of the cards here still says `Model Management` instead of `Managing Infrastructure`.

Reply:
😈